docs/grid_exchange_formats/cgmes/export.md

```diff
@@ -1,8 +1,154 @@
+
+When exporting, it is verified that the main network and all subnetworks have the same scenario time (network case date). If they are different, an error is logged.
+
+If no version is provided as a parameter for the exported files, the output version is determined based on the max version of input CGM SV metadata and IGM SSH versions.
+
+If the dependencies have to be updated automatically (see parameter **iidm.export.cgmes.update-dependencies** below), the exported instance files will contain metadata models where:
+* Updated SSH for IGMs supersede the original ones.
+* Updated SV for the CGM depends on the updated SSH from IGMs and on the original TP from IGMs.
+
+The filenames of the exported instance files will follow the pattern:
+* For the CGM SV: `<basename>_SV.xml`.
+* For the IGM SSHs: `<basename>_<IGM name>_SSH.xml`. The IGM name is built from the country code of the first substation or the IIDM name if no country is present.
+
+The basename is determined from the parameters, or the basename of the export data source or the main network name.
+
+As an example, you can export one of the test configurations that has been provided by ENTSO-E. It is available in the cgmes-conformity module of the powsybl-core repository. If you run the following code:
+
+```java
+Network cgmNetwork = Network.read(CgmesConformity1Catalog.microGridBaseCaseAssembled().dataSource());
+
+Properties exportParams = new Properties();
+exportParams.put(CgmesExport.PROFILES, List.of("SV", "SSH"));
+exportParams.put(CgmesExport.EXPORT_BOUNDARY_POWER_FLOWS, true);
+exportParams.put(CgmesExport.NAMING_STRATEGY, "cgmes");
+exportParams.put(CgmesExport.CGM_EXPORT, true);
+exportParams.put(CgmesExport.UPDATE_DEPENDENCIES, true);
+exportParams.put(CgmesExport.MODELING_AUTHORITY_SET, "MAS1");
+
+cgmNetwork.write("CGMES", exportParams, new FileDataSource(Path.of("/exampleFolder"), "exampleBase"));
+```
+
+You will obtain the following files in your `exampleFolder`:
+
+```
+exampleBase_BE_SSH.xml
+exampleBase_NL_SSH.xml
+exampleBase_SV.xml
+```
+
+where the updated SSH files will supersede the original ones, and the SV will contain the correct dependencies of new SSH and original TPs.
+
+## CGM (Common Grid Model) manual export
+
+If you want to intervene in how the updated IGM SSH files or the CGM SV are exported, you can make multiple calls to the CGMES export function.
+
+You can use following code for reference:
+
+```java
+Network cgmNetwork = Network.read(CgmesConformity1Catalog.microGridBaseCaseAssembled().dataSource());
+
+// We decide which version we want to export
+int exportedVersion = 18;
+
+// Common export parameters
+Properties exportParams = new Properties();
+exportParams.put(CgmesExport.EXPORT_BOUNDARY_POWER_FLOWS, true);
+exportParams.put(CgmesExport.NAMING_STRATEGY, "cgmes");
+// We do not want a quick CGM export
+exportParams.put(CgmesExport.CGM_EXPORT, false);
+exportParams.put(CgmesExport.UPDATE_DEPENDENCIES, false);
```

Comment:
Are you sure update dependencies parameter is used for CGM export?

Reply:
yes, the update dependencies parameter is considered for both for CGM quick export and in manual network export. For CGM quick export dependencies are updated in `CgmesExport::updateDependenciesCGM`. For manual network export they are handled in `CgmesExport::updateDependenciesIGM`.